src/backend/distributed/commands/citus_version.c

```diff
@@ -0,0 +1,36 @@
+/*-------------------------------------------------------------------------
+ *
+ * citus_version.c
+ *	  Routines related to the version of the Citus binaries
+ *
+ * Copyright (c) 2012-2016, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#include "postgres.h"
+
+#include "fmgr.h"
+
+#include "citus_version.h"
+#include "utils/builtins.h"
+
+
+/* exports for SQL callable functions */
+PG_FUNCTION_INFO_V1(citus_running_version);
+
+#if !defined(CITUS_BINARY_VERSION)
```

Comment:
The thing after the `-` character is not the patch version, it's the catalog version for that minor version.

The patch version is the `1` in `6.0.1`. The catalog version is the `14` in `6.1-14`.